amf-rdf/shared/src/main/scala/amf/rdf/client/scala/RdfConfiguration.scala

```diff
@@ -0,0 +1,29 @@
+package amf.rdf.client.scala
+
+import amf.core.client.scala.AMFGraphConfiguration
+import amf.core.client.scala.config.AMFOptions
+import amf.core.client.scala.errorhandling.DefaultErrorHandlerProvider
+import amf.core.internal.annotations.serializable.CoreSerializableAnnotations
+import amf.core.internal.convert.CoreRegister
+import amf.core.internal.entities.CoreEntities
+import amf.core.internal.plugins.document.graph.entities.DataNodeEntities
+import amf.core.internal.registries.AMFRegistry
+import amf.core.internal.resource.AMFResolvers
+import amf.core.internal.transform.pipelines.BasicTransformationPipeline
+import amf.rdf.internal.plugins.{RdfParsePlugin, RdfRenderPlugin, RdfSyntaxPlugin, RdfSyntaxRenderPlugin}
+
+object RdfConfiguration {
+  def apply(): AMFGraphConfiguration = {
```

Comment:
we can probably just do:
`AMFGraphConfiguration.predefined().withPlugins(.....)`

Reply:
to avoid having to copy AMFGraphConfiguration setup